### PR TITLE
docs(theme): Shorten tables in README, and update markdown

### PR DESCRIPTION
**Pull request by lynnjepsen**
The markdown needs to work in our internal Google docs as well. The changes in markdown doesn't make any visual changes, it just makes it easier to copy/paste into our internal tool later.

However, shortening the tables makes the README easier to...well...read.

### CODE REVIEW

File: packages/mdc-theme/README.md [hunk 1]
```diff
@@ -79,130 +71,61 @@ $mdc-theme-background: #fff;
...
+#### CSS Custom Properties
+
+CSS Custom property | Description
+--- | ---
+`--mdc-theme-primary` | The theme primary color
+`--mdc-theme-accent` | The theme accent color
+`--mdc-theme-background` | The theme background color
+`--mdc-theme-text-<TEXT_STYLE>-on-<THEME_COLOR>` | Text color for TEXT_STYLE on top of THEME_COLOR background
+`--mdc-theme-text-<TEXT_STYLE>-on-light` | Text color for TEXT_STYLE on top of light background
+`--mdc-theme-text-<TEXT_STYLE>-on-dark` | Text color for TEXT_STYLE on top of dark background
```
Comment (amsheehan) [inline packages/mdc-theme/README.md]: Add a note:

> **NOTE:** Where <TEXT_STYLE> is directly correlated to the typographical construct the component implements. E.g.: `mdc-theme-text-hint-on-dark`

Remember our users in some cases will be new to web development and I can't tell you how long it took me to figure out what capital letters inside angle brackets meant when I was a wee lad just starting out. 😂 😭

File: packages/mdc-theme/README.md [hunk 2]
```diff
@@ -79,130 +71,61 @@ $mdc-theme-background: #fff;
...
+CSS Class | Description
+--- | ---
+`mdc-theme--primary` | Sets the text color to the theme primary color
+`mdc-theme--accent` | Sets the text color to the theme accent color
+`mdc-theme--background` | Sets the background color to the theme background color
+`mdc-theme--primary-bg` | Sets the background color to the theme primary color
+`mdc-theme--accent-bg` | Sets the background color to the theme accent color
+`mdc-theme--text-<TEXT_STYLE>-on-<THEME_COLOR>` | Sets text to a suitable color for TEXT_STYLE on top of THEME_COLOR background
+`mdc-theme--text-<TEXT_STYLE>-on-light` | Sets text to a suitable color for TEXT_STYLE on top of light background
+`mdc-theme--text-<TEXT_STYLE>-on-dark` | Sets text to a suitable color for TEXT_STYLE on top of dark background
```
Comment (amsheehan) [inline packages/mdc-theme/README.md]: I would duplicate the note above. People may be jumping down to this before they read about typography.

File: packages/mdc-theme/README.md [hunk 3]
```diff
@@ -79,130 +71,61 @@ $mdc-theme-background: #fff;
...
-| `text-disabled-on-dark`        | Disabled text on top of a dark-colored background.         |
-| `text-icon-on-dark`            | Icons on top of a dark-colored background.                 |
+Property Name | Description
+--- | ---
+`primary` | The theme primary color
+`accent` | The theme accent color
+`background` | The theme background color
+`text-<TEXT_STYLE>-on-<THEME_COLOR>` | TEXT_STYLE on top of THEME_COLOR background
+`text-<TEXT_STYLE>-on-light` | TEXT_STYLE on top of a light background
+`text-<TEXT_STYLE>-on-dark` | TEXT_STYLE on top of a dark background
```
Comment (amsheehan) [inline packages/mdc-theme/README.md]: Aaaaand here as well.

File: packages/mdc-theme/README.md (hunk 1 above)
Comment (lynnjepsen) [inline packages/mdc-theme/README.md]: I added a note...but I rewrote it to be a little more explicit.

File: packages/mdc-theme/README.md (hunk 2 above)
Comment (lynnjepsen) [inline packages/mdc-theme/README.md]: Done

File: packages/mdc-theme/README.md (hunk 3 above)
Comment (lynnjepsen) [inline packages/mdc-theme/README.md]: Done

**Comment by lynnjepsen** [comment]
Added the explicit notes about uppercase angled brackets format. PTAL

**Pull request merged**